api/db-cdr/prep-tables/make-prep-survey.py

```diff
@@ -0,0 +1,470 @@
+import argparse
+import csv
+import os
+import shutil
+from google.cloud import bigquery
+from google.cloud import storage
+from io import StringIO
+from os import listdir
+from os.path import isfile, join
+
+# we will increment this as the row number as we write each row in each output file
+id_controlled = 1
+id_registered = 1
+id_all = 1
+
+home_dir = "../csv"
+
+# key = csv prefix, value = csv output file name
+surveys = {"ENGLISHBasics_DataDictionary": "Basics",
+           "ENGLISHHealthCareAccessUtiliza_DataDictionary": "HealthCareAccessUtiliza",
+           "ENGLISHLifestyle_DataDictionary": "Lifestyle",
+           "ENGLISHOverallHealth_DataDictionary": "OverallHealth",
+           "ENGLISHPersonalMedicalHistory_DataDictionary": "PersonalMedicalHistory"}
+
+# field annotation flags
+annotation_flags = {"REGISTERED_TOPIC_SUPPRESSED": "0",
+                    "REGISTERED_QUESTION_SUPPRESSED": "0",
+                    "REGISTERED_ANSWERS_BUCKETED": "0",
+                    "REGISTERED_ANSWER_SUPPRESSED": "0",
+                    "CONTROLLED_TOPIC_SUPPRESSED": "0",
+                    "CONTROLLED_QUESTION_SUPPRESSED": "0",
+                    "CONTROLLED_ANSWERS_BUCKETED": "0",
+                    "CONTROLLED_ANSWER_SUPPRESSED": "0"}
+
+# these are the column names for the controlled and registered output files
+headers = ['id', 'parent_id', 'code', 'name', 'type', 'min', 'max',
+           'answers_bucketed']
+
+# these are the headers for outputAll file
+headersAll = ['id', 'parent_id', 'code', 'name', 'type', 'min', 'max',
+              'registered_topic_suppressed', 'registered_question_suppressed',
+              'registered_answer_bucketed', 'registered_answer_suppressed', 'controlled_topic_suppressed',
+              'controlled_question_suppressed', 'controlled_answer_bucketed', 'controlled_answer_suppressed']
+
+
+def main():
```

Comment:
Whenever you see these codacy messages like the one below. This is telling you that you need to fix something in your code. It will not allow you to merge your PR without fixing these issues. In this case it's telling you that your main function is too complex.